app/controllers/comments_controller.rb

```diff
@@ -221,6 +221,11 @@ def hide
     authorize @comment
     @comment.hidden_by_commentable_user = true
     @comment&.commentable&.update_column(:any_comments_hidden, true)
+
+    Notification.destroy_by(user_id: current_user.id,
+                            notifiable_type: "Comment",
```

Comment:
Hello there,
  I just saw this code segment in **app/models/comment.rb** thanks to this comment https://forem.dev/msarit/comment/f8 thanks to @msarit 

```ruby
  after_update :update_descendant_notifications, if: :deleted
  after_update :remove_notifications, if: :deleted
```

Is it possible that I can write another after_update to remove notifications if it is hidden making the code more clean?

Reply:
I just remembered this and commented here. I'll create separate Issue if it is needed.

Reply:
I'll create a separate issue for this :)